cmd/snap/cmd_download.go

```diff
@@ -94,7 +94,7 @@ func fetchSnapAssertionsDirect(tsto *tooling.ToolingStore, snapPath string, snap
 	}
 	f := tsto.AssertionFetcher(db, save)
 
-	_, err = image.FetchAndCheckSnapAssertions(snapPath, snapInfo, nil, f, db)
+	_, err = image.FetchAndCheckSnapAssertions(snapPath, snapInfo, nil, nil, f, db)
```

Comment:
should we have a TODO here about supporting downloading components ?

Reply:
TODO added